### PR TITLE
Add Istio UIs to module

### PR DESCRIPTION
**Pull request by barchw**
**Description**

Changes proposed in this pull request:

- Add Istio UIs to module

### CODE REVIEW

File: tests/ui/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Prerequisites
+
+If you want to use an existing cluster, you must copy your cluster's kubeconfig file to `fixtures/kubeconfig.yaml`.
+
+## Installation
+
+To install the dependencies, run the `npm clean install` command.
+
+## Test development using Headless mode with Chrome browser
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:15]:
```suggestion
## Test Development Using Headless Mode with Chrome Browser
```
We've changed the guidelines to use title case for headings. I updated Istio docs but haven't updated API Gateway yet

File: tests/ui/tests/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### Using a local Kyma Dashboard instance
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:17]:
```suggestion
### Using a Local Kyma Dashboard Instance
```

File: tests/ui/tests/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+npm run start-k3d
+```
+
+Start the local Kyma Dashboard instance:
+
+```bash
+npm run start-dashboard
+```
+
+#### Run tests
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:31]:
```suggestion
#### Run Tests
```

File: tests/ui/tests/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+```bash
+npm run test
+```
+
+#### Run Cypress UI tests in the test runner mode
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:37]:
```suggestion
#### Run Cypress UI Tests in the Test Runner Mode
```

File: tests/ui/tests/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+```bash
+npm run start
+```
+
+### Using a remote Kyma Dashboard instance
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:43]:
```suggestion
### Using a Remote Kyma Dashboard Instance
```

File: tests/ui/tests/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+#### Optional: Log in to a cluster using OIDC
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:45]:
```suggestion
#### Optional: Log In to a Cluster Using OIDC
```

File: tests/ui/tests/README.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+If a cluster requires OIDC authentication, include the additional arguments **CYPRESS_OIDC_PASS** and **CYPRESS_OIDC_USER** while running the npm scripts.
+
+#### Run tests
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:49]:
```suggestion
#### Run Tests
```

File: tests/ui/tests/README.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+```bash
+CYPRESS_OIDC_PASS={YOUR_PASSWORD} CYPRESS_OIDC_USER={YOUR_USERNAME} CYPRESS_DOMAIN={REMOTE_CLUSTER_DASHBOARD_DOMAIN} npm run test
+```
+
+#### Run Cypress UI tests in the test runner mode
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:54]:
```suggestion
#### Run Cypress UI Tests in the Test Runner Mode
```

File: tests/ui/tests/README.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+```bash
+CYPRESS_OIDC_PASS={YOUR_PASSWORD} CYPRESS_OIDC_USER={YOUR_USERNAME} CYPRESS_DOMAIN={REMOTE_CLUSTER_DASHBOARD_DOMAIN} npm run start
+```
+
+## Run tests in Continuous Integration System
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:60]:
```suggestion
## Run Tests in Continuous Integration System
```

File: tests/ui/tests/README.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
+# Tests
+
+## Overview
+
+This project contains API Gateway integration tests for Kyma Dashboard.
```
Comment (nataliasitko) [inline tests/ui/tests/README.md:5]:
```suggestion
This project contains API Gateway integration tests for Kyma dashboard.
```

File: tests/ui/tests/scripts/k3d-local-dev.sh [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+set -ex
+
+echo "Provisioning k3d cluster"
+kyma provision k3d
+
+export KUBECONFIG=$(k3d kubeconfig merge kyma)
+
+echo "Apply istio"
+kubectl apply -f https://github.com/kyma-project/istio/releases/latest/download/istio-manager.yaml
```
[the remark this answers is not in the document]
Comment (barchw) [reply tests/ui/tests/scripts/k3d-local-dev.sh]: I will leave it, only adding the creation of `kyma-system` namespace. I think that it should by default deploy the latest version, if there is a need to use a different one, the approach from `k3d-ci-kyma-dashboard-integration.sh` could be used.

**Review by kolodziejczak — APPROVED** [review]

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**